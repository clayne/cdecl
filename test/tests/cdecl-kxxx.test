cdecl @ @ -k xxx @ @ 64
