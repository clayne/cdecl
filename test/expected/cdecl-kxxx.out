cdecl: "xxx": invalid value for --color/-k; must be one of: always, auto, isatty, never, not_file, not_isreg, tty
